solidjs-tailwind/src/pages/Home.jsx

```diff
@@ -1,15 +1,33 @@
+import { createResource } from 'solid-js';
 import { useAuth } from '../auth';
+import { useOctokit } from '../github';
+import { TopRepos } from '../components/TopRepos';
+import UserGists from '../components/UserGists/UserGists';
 
 const Home = () => {
-const { authStore } = useAuth();
+  useAuth().preventUnauthorised();
 
+  const [data] = createResource(() => {
+    try {
+      return useOctokit()
+        .rest.users.getAuthenticated()
+        .then((response) => response.data);
+    } catch {
+      return Promise.resolve({});
+    }
+  });
```

Comment:
@AllStackDev1 mentioned in the last PR but it looks like we're using the gql api with the fetch util for all the data fetching in this starter kit. We might want to get rid of Ocotokit if thats the case

Reply:
yep @danecando that's right, we should get rid of the Octokit, also the PR was rebased within `main` to have the latest commits.